static/config.js

```diff
@@ -10,12 +10,17 @@ const DEBUG_OUTPUT = true;
 const TIME_SYNC_DELTA = 60000;
 const DISCONNECT_TIMEOUT = 5000;
 const ERROR_TIMEOUT = 4000;
+const GAME_BUTTON_COOLDOWN_TIME = 200;
 const DEFAULT_SETTINGS = {
     wordNumber: 100,
     turnsNumber: 10
 }
-const GAME_BUTTON_COOLDOWN_TIME = 200;
-
+const ERROR_MSGS = {
+	101: "Нужно ввести ключ комнаты",
+	102: "Нужно ввести имя",
+	103: "Такое имя уже занято",
+	104: "Чтобы войти в игру нужно ввести имя, с которым вы играли."
```

Comment:
Ну не надо менять ключи в `*.po`, не меняя места, где они используются.
```suggestion
	104: "Чтобы войти в игру, нужно ввести имя, с которым вы играли."
```

Reply:
Это надо поправить, так как пока это приводит к русскому сообщению при включенном английском языке. Все ли согласны с данным исправлением?